src/components/CodeHighlight/CodeHighlight.js

```diff
@@ -0,0 +1,61 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+
+import * as Languages from './Languages';
+import * as Themes from './Themes';
+
+const { LANGUAGES } = Languages;
+const {
+  THEMES,
+  themes,
+} = Themes;
+
+const markup = val => {
+  return { __html: val };
+};
+
+export const CodeHighlight = memo(props => {
+  const {
+    code,
+    language,
+    theme,
+    usePre,
+    className,
+  } = props;
+
+  const hlTheme = themes[theme];
+
+  if (usePre) {
+    return (
+      <pre className={theme}>
+        <code
+          className={className}
+          dangerouslySetInnerHTML={markup(Languages[language](code, hlTheme))}
+        />
+      </pre>
+    );
+  }
+
+  return (
+    <code
```

Comment:
Seems like this snippet is the same in as above. Please store it to variable in order to use it only once.

Reply:
Done